apps/home/app/@readme/people/page.tsx

```diff
@@ -0,0 +1,15 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import { Readme } from '@/_shared'
+
+export default async function Page({
+  searchParams,
+}: {
+  searchParams: { username: string }
+}) {
+  const { username } = searchParams
+  const filePath = path.join(process.cwd(), 'app/@readme/_shared/content', `${username}.mdx`)
```

Comment:
```suggestion
  const filePath = path.join(process.cwd(), 'app/@readme/_shared/content', `${searchParams.username}.mdx`)
```

한 번 밖에 안 쓰이는 변수인데, 구조분해 할당 할 필요가 있을까요 ? 🤔